src/ResourceManager/Compute/Commands.Compute/Models/AzureDiskEncryptionStatusContext.cs

```diff
@@ -22,6 +22,7 @@ enum EncryptionStatus
         Encrypted,
         NotEncrypted,
         NotMounted,
+        DecryptionInProgress,
```

Comment:
@cormacpayne Yes the change is related to encryption disable. I can create a separate PR for this one line but it is needed.